editor/editor_config.hpp

```diff
@@ -5,7 +5,7 @@
 #include <string>
 #include <vector>
 
-#include "3party/pugixml/pugixml/src/pugixml.hpp"
+#include "pugixml.hpp"
```

Comment:
There is no such file in the current directory, it leads to unnecessary `stat` calls by the compiler. `<>` should be used instead.